src/test/utils.js

```diff
@@ -80,6 +80,15 @@ export const stripWhitespace = (str: string) =>
     .replace(/([;\{\}])/g, '$1  ')
     .replace(/\s+/g, ' ')
 
+export const getCSS = (scope?: Document | HTMLElement = document) => {
```

Comment:
Just a question: 

Won't Flow infer `function | object` and let undefined pass for `getCSS`? So wouldn't you get an error if anything but an existing `document` object is passed in? I realize this is just a test and you're completely controlling the execution-- I'm just curious.

Reply:
Changed it to be more explicit about the scope. The current typing appears to be necessary though as `document` is not compatible to `HTMLElement`